.binder/environment.yml

```diff
@@ -3,28 +3,25 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - numpy==1.18.4
-  - mkl=2020.0=166
-  - mkl_fft=1.0.15=py36ha843d7b_0
-  - pip==20.1
-  - python=3.6.10=hcf32534_1
-  - scipy=1.4.1=py36h0b6359f_0
-  - six=1.14.0=py36_0
+  - python=3.9.5
+  - numpy==1.21.0
+  - scipy==1.7.0
+  - pip=21.1.2
+  - six==1.16.0
+  - holoviews==1.13.2
   - pip:
-    - bokeh==2.0.2
+    - bokeh==2.3.2
     - fissa[plotting]
     - future==0.18.2
-    - h5py==2.10.0
-    - holoviews==1.13.2
-    - imagecodecs==2020.2.18
-    - imageio==2.8.0
-    - numba==0.49.0
-    - pandas==1.0.3
-    - pillow==7.1.2
-    - read-roi==1.5.2
-    - scikit-image==0.16.2
-    - scikit-learn==0.22.2.post1
-    - shapely==1.7.0
-    - sima==1.3.2
-    - suite2p==0.7.1
-    - tifffile==2020.2.16
+    - h5py==3.3.0
+    - imageio==2.9.0
+    - numba==0.53.1
+    - pandas==1.2.5
+    - pillow==8.2.0
+    - read-roi==1.6.0
+    - scikit-image==0.18.1
+    - scikit-learn==0.24.2
+    - shapely==1.7.1
+    - suite2p==0.10.0
+    - tifffile==2021.6.14
+    - pyqt5==5.15.4
```

Comment:
Alphabetise
```suggestion
    - pyqt5==5.15.4
    - read-roi==1.6.0
    - scikit-image==0.18.1
    - scikit-learn==0.24.2
    - shapely==1.7.1
    - suite2p==0.10.0
    - tifffile==2021.6.14
```

Reply:
> Some commits ([4de477d](https://github.com/rochefort-lab/fissa/commit/4de477d8013d361de6023e53e145efb9305cabec) and [9d5e34b](https://github.com/rochefort-lab/fissa/commit/9d5e34b16a44344e2d479a88cb6c868dec9b46bb)) are undoing formatting style fixes I made to the suite2p and SIMA notebooks. I would appreciate if these commits were dropped from the PR. I expect this has come from checking out the new branch, using your notebooks that were forked from the previous (old) copies of these notebooks, overwriting the notebooks, and then committing everything in the notebook without checking what it is that you're committing. If you're actually including these changes on purpose, please let me know.

I did do these in purpose I think, in that the pre-commit made these changes and I thought they would be sensible.